examples/wrapper-components/react-javascript/src/App.js

```diff
@@ -1,32 +1,12 @@
 import './App.css';
-
-
-import { IfxAccordion, IfxAccordionItem } from '@infineon/infineon-design-system-react';
-
+import Button from './components/button/button';
 
```

Comment:
We do not want to import the Button from the components folder here, but use the exported IfxButton from our wrapper package

Reply:
Why? 
This is the App.js file. This is the file that gets shown in the browser when you run your React app. The IfxButton component is imported and used in the Button component file in React, and then in App.js, we import that React component , and we render it. This is imply used to render the React Button.

Reply:
and we do, inside the components/button folder, where I create button.js file, inside of which I import IfxButton. This is how we create a React component, and then to render it on the browser, we import it inside the App.js. You do it the same way in Vue. You have /components folder, and then you have the App.vue, where you import and render Vue components, which use our Stencil components.